### PR TITLE
ENT-4016: Test that preserve false in body copy_from does not modify permissions

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @vpodzime can review this?

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/1023/)

https://ci.cfengine.com/job/pr-pipeline/1023/

File: tests/acceptance/10_files/copy_from_preserve_false.cf.src [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This file is used as part of a test. Its hashed and compared against another
+file.
```
Comment (olehermanse) [inline tests/acceptance/10_files/copy_from_preserve_false.cf.src]: Add a newline?

Comment (nickanderson) [reply tests/acceptance/10_files/copy_from_preserve_false.cf.src]: As you wish.

**Pull request merged**